.github/workflows/ci.yml

```diff
@@ -220,6 +221,27 @@ jobs:
           make VERBOSE=1 -j${BUILD_CONCURRENCY} ${make_targets}
           ctest --timeout ${TEST_TIMEOUT} --output-on-failure ${ctest_flags}
 
+  dr-testing:
+    name: Run DR shp-tests
```

Comment:
Can we align this job's name with the others?

Reply:
this job was moved to be part of linux-testing